File: app/src/main/java/com/example/myapplication/Images.java
package com.example.myapplication;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Images {

    @SerializedName("original")
    @Expose
    private Original original;

    public Original getOriginal() {
        return original;
    }

    public void setOriginal(Original original) {
        this.original = original;
    }

    @Override
    public String toString() {
        return "Images{" +
                "original=" + original +
                '}';
    }
}

File: app/src/main/java/com/example/myapplication/model/Images.kt
package com.example.myapplication.model

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import com.example.myapplication.model.Original


data class Images(
    @SerializedName("original")
    @Expose
    var original: Original? = null
)